src/components/Navigation/Navigation.js

```diff
@@ -40,6 +42,9 @@ const Navigation = props => {
             {/* <Link to="/new-topic"> */}
```

Comment:
Please remove any commented out code from main. It doesn't have a place in a production environment.